src/common/transformers/UPPERCASE.js

```diff
@@ -1,27 +1,9 @@
-import smartMap from './smartMap';
+import { transformer } from './transformer';
 
-export const valueToUpperCase = value => {
-    if (!value) {
-        return null;
-    }
+export const valueToUpperCase = value =>
+    typeof value === 'string' ? value.toUpperCase() : value;
```

Comment:
Here too, we have to update [the documentation](https://www.gitbook.com/book/lodex/lodex-user-documentation/edit#/edit/master/Administration/Modèle/Transformers/UPPERCASE.md).